src/ComputeSharp.D2D1/Shaders/Interop/Factories/D2D1AffineTransformMapperFactory.cs

```diff
@@ -0,0 +1,112 @@
+﻿using System;
+using System.Numerics;
+using ComputeSharp.D2D1.Interop;
+using ComputeSharp.D2D1.Shaders.Interop.Factories.Abstract;
+using ComputeSharp.D2D1.Shaders.Interop.Helpers;
+
+namespace ComputeSharp.D2D1.Shaders.Interop.Factories;
+
+/// <summary>
+/// A custom <see cref="D2D1TransformMapperFactory{T, TSelf, TParameters, TTransformMapper}"/> implementation for an affine transform.
+/// </summary>
+/// <typeparam name="T">The type of D2D1 pixel shader associated to the transform mapper.</typeparam>
+internal sealed class D2D1AffineTransformMapperFactory<T> : D2D1TransformMapperFactory<T, D2D1AffineTransformMapperFactory<T>, Matrix3x2, D2D1AffineTransformMapperFactory<T>.TransformMapper>
+    where T : unmanaged, ID2D1PixelShader
+{
+    /// <summary>
+    /// A <see cref="D2D1TransformMapperParametersAccessor{T, TParameters}"/> implementation for a fixed affine transform matrix.
+    /// </summary>
+    public sealed class FixedMatrix : D2D1TransformMapperParametersAccessor<T, Matrix3x2>
+    {
+        /// <summary>
+        /// Gets the fixed affine transform matrix.
+        /// </summary>
+        public Matrix3x2 Amount { get; init; }
+
+        /// <inheritdoc/>
+        public override Matrix3x2 Get(in T shader)
+        {
+            return Amount;
+        }
+    }
+
+    /// <summary>
+    /// A <see cref="D2D1TransformMapperParametersAccessor{T, TParameters}"/> implementation for a dynamic affine transform matrix.
+    /// </summary>
+    public sealed class DynamicMatrix : D2D1TransformMapperParametersAccessor<T, Matrix3x2>
+    {
+        /// <summary>
+        /// Gets the <see cref="D2D1TransformMapperFactory{T}.Accessor{TResult}"/> instance to get the dynamic affine transform matrix.
+        /// </summary>
+        public D2D1TransformMapperFactory<T>.Accessor<Matrix3x2>? Accessor { get; init; }
+
+        /// <inheritdoc/>
+        public override Matrix3x2 Get(in T shader)
+        {
+            return Accessor!(in shader);
+        }
+    }
+
+    /// <summary>
+    /// A custom <see cref="D2D1TransformMapper{T, TParameters}"/> implementation for an inflate transform.
```

Comment:
copypasta bug? "... for an _inflate_ transform"

Reply:
Yup! Thanks, fixed 😄